app/packages/app/src/pages/datasets/DatasetPage.tsx

```diff
@@ -75,6 +76,8 @@ const DatasetPageQueryNode = graphql`
 const DatasetPage: Route<DatasetPageQuery> = ({ prepared }) => {
   const data = usePreloadedQuery(DatasetPageQueryNode, prepared);
   const isModalActive = Boolean(useRecoilValue(fos.isModalActive));
+  const count = useRecoilValue(fos.datasetSampleCount);
```

Comment:
cool, so this is not based on for example filters returning 0 samples but actual sample count of a dataset

Reply:
Indeed. This count should not factor in filters applied.